packages/docs/src/routes/community/menu.md

```diff
@@ -5,5 +5,5 @@
 - [Groups](groups/index.mdx)
 - [Projects](projects/index.mdx)
 - [Values](values/index.mdx)
-- [Media](../media/index.tsx)
-- [Showcase](../showcase/index.tsx)
+- [Media](/community/media/)
```

Comment:
can we adjust them so they point as well to the `.mdx`  file?

Reply:
The thing is, it's not an `.mdx` file (`.tsx`) 😅